Scripts/Views/Settings.gd

```diff
@@ -115,6 +115,10 @@ func _on_Option_item_selected(index: int) -> void:
 
 func _on_secsDashboard_toggled(button_pressed: bool) -> void:
 	res.show_secs_dash = button_pressed
+	
```

Comment:
You'll also need to initialize the default value in the set_up_btns() function of this script.
(somewhere around line 35,36)